common/src/java/org/apache/hive/http/ProfileServlet.java

```diff
@@ -344,10 +357,11 @@ private Output getOutput(final HttpServletRequest req) {
       try {
         return Output.valueOf(outputArg.trim().toUpperCase());
       } catch (IllegalArgumentException e) {
-        return Output.SVG;
+        LOG.warn("Output format value is invalid, returning with default HTML");
```

Comment:
Can you add what value is actually configured in the log as well
```
        LOG.warn("Output format value {} is invalid, returning with default HTML", outputArg);
```

Reply:
makes sense!